tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -110,6 +110,92 @@
       gst_harness_teardown (h); \
     }
 
+#define GET_MODEL_PATH(model_name) \
+  const gchar *root_path = g_getenv ("NNSTREAMER_BUILD_ROOT_PATH"); \
+  gchar *test_model, *str_launch_line; \
+  if (root_path == NULL) \
+    root_path = ".."; \
+  test_model = g_build_filename (root_path, "tests", "test_models", "models", \
+    #model_name, NULL); \
+  ASSERT_TRUE (g_file_test (test_model, G_FILE_TEST_EXISTS)); \
```

Comment:
```suggestion
  ASSERT_TRUE (g_file_test (test_model, G_FILE_TEST_EXISTS));
```

Reply:
> IMO, multi-line defined macro is better to be wrapped by do{ } while(0);

ditto. unless there is a scoping problem.